src/views/Main/Catalog/Components/EntityTypeList.js

```diff
@@ -65,7 +67,31 @@ export class EntityTypeList extends React.Component {
   updateFn = () => {
     EntityDataModelApi.getAllEntityTypes()
       .then((entityTypes) => {
-        this.setState({ entityTypes: Utils.addKeysToArray(entityTypes) });
+        EntityDataModelApi.getAllPropertyTypes()
+        .then((propertyTypes) => {
+          const allPropNames = {};
```

Comment:
what is the purpose of `allPropNames` and `allPropNamespaces`? it looks like you're trying to achieve a bi-map. why do you need a bi-map?